src/Routes.tsx

```diff
@@ -14,12 +14,41 @@ import { CommonRoutesFeatures } from './RoutesWrap';
 const { MainView } = lazily(() => import('~views/Main/Main'));
 const { Reports } = lazily(() => import('~views/Reports/Reports'));
 const { ReportPage } = lazily(() => import('~views/Report/Report'));
+const { AboutPage } = lazily(() => import('~views/About/About'));
 const { BivariateManagerPage } = lazily(
   () => import('~views/BivariateManager/BivariateManager'),
 );
 
 export function RoutedApp() {
   const [{ data: userModel, loading }] = useAtom(userResourceAtom);
+
+  function isFirstTimeVisit() {
+    const firstTimeVisit =
+      userModel &&
+      !loading &&
+      location.pathname == '/' &&
```

Comment:
it feels like you need to use config.baseUrl here instead of '/', when you deploy it will be /active instead of / comparing to localhost

Reply:
@vkozio /active/ added on nginx side, and js starts loading after.
You can check this if put breakpoint in disaster ninja Routes.tsx file

Reply:
config.baseUrl is dynamic - / on locahost, /active - on prod

Reply:
the DN2 app lives under /active/, the DN1 app lives under /live/, in the / we have just a redirector html to route to current best version.